### PR TITLE
add commands for shower, small fixes

### PR DESCRIPTION
**Pull request by MichaelDvP**
Formatting was broken for `F_(switch)`, interpreted as keyword, so i changed the name to switcher, but still outputs "switch"  for the device.

### CODE REVIEW

**Review by proddy — APPROVED** [review]

File: src/mqtt.cpp [hunk 1]
```diff
@@ -984,6 +984,7 @@ bool Mqtt::publish_ha_sensor_config(uint8_t               type,        // EMSdev
             break;
         default:
             // plain old sensor
+            snprintf(topic, sizeof(topic), "sensor/%s", config_topic);
```
[the remark this answers is not in the document]
Comment (proddy) [reply src/mqtt.cpp:987]: @MichaelDvP fix this and then we can merge the PR

[the remark this answers is not in the document]
Comment (MichaelDvP) [reply src/mqtt.cpp:987]: No, it is not created later in default case, leading to 3 sec mqtt restarts for tp1de. ((btw. there is also nothing like readonly_sensors).  
We have all types in the case statements, so default is not used and do no harm. But for the timers i've created a new type and not adding to the case statements gives scrambled topics.

[the remark this answers is not in the document]
Comment (proddy) [reply src/mqtt.cpp:987]: the readonly_sensors was in my code, but not checked-in yet. I'll merge this PR and apply the fix afterwards.

**Comment by proddy** [comment]
build broken - standalone needs updating

**Pull request merged**